From 654f197f1965d5be52b96c2f1af27d8bc96f9c78 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Tue, 10 Feb 2015 22:21:47 -0800
Subject: [PATCH 2/2] darwin: build fix

xfont.c:237:14: error: implicit declaration of function 'GetGLXDRIDrawable' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
   glxdraw = GetGLXDRIDrawable(CC->currentDpy, CC->currentDrawable);
             ^
Fixes regression from 291be28476ea60c6fb1eb2a882e2e25def5d3735

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit e68b67b53fce39a8c93188262d9e795ca50750ac)
---
 src/glx/xfont.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/glx/xfont.c b/src/glx/xfont.c
index a086b7a..00498bc 100644
--- a/src/glx/xfont.c
+++ b/src/glx/xfont.c
@@ -221,7 +221,10 @@ DRI_glXUseXFont(struct glx_context *CC, Font font, int first, int count, int lis
    XGCValues values;
    unsigned long valuemask;
    XFontStruct *fs;
+
+#if !defined(GLX_USE_APPLEGL)
    __GLXDRIdrawable *glxdraw;
+#endif
 
    GLint swapbytes, lsbfirst, rowlength;
    GLint skiprows, skippixels, alignment;
@@ -234,9 +237,11 @@ DRI_glXUseXFont(struct glx_context *CC, Font font, int first, int count, int lis
    dpy = CC->currentDpy;
    win = CC->currentDrawable;
 
+#if !defined(GLX_USE_APPLEGL)
    glxdraw = GetGLXDRIDrawable(CC->currentDpy, CC->currentDrawable);
    if (glxdraw)
       win = glxdraw->xDrawable;
+#endif
 
    fs = XQueryFont(dpy, font);
    if (!fs) {
-- 
2.3.0

